src/components/Results.tsx

```diff
@@ -37,6 +43,42 @@ const StyledContainer = styled(Box)(({ theme }) => ({
 }));
 
 export const Results: React.FC = () => {
+  const theme = useTheme();
+  const showResultsNumber = useMediaQuery(theme.breakpoints.down('md'));
+
+  const [view, setView] = useState({});
```

Comment:
i would rename it somehow so it is clear that this is a css object.

Reply:
fixed